.editorconfig

```diff
@@ -5,18 +5,7 @@ indent_style = space
 end_of_line = lf
 trim_trailing_whitespace = true
 insert_final_newline = true
-
-[*.py]
-max_line_length = 79
-indent_size = 2
-
-[*.rst]
-max_line_length = 79
 indent_size = 2
 
-[*.md]
-max_line_length = 79
-indent_size = 2
-
-[*.yml]
-indent_size = 2
+[*.py]
+max_line_length = 88
```

Comment:
We use 80 internally. Can we match that?

Otherwise the code internally and externally looks different and that's not ideal.

Reply:
I'm just following what's showing in `pyproject.toml`:
```toml
[tool.ruff]
preview = true
exclude = [
    ".git",
    "build",
    "__pycache__",
]
line-length = 88
indent-width = 2
target-version = "py310"
```
Should that be changed?

Reply:
Okay, looking deeper, I actually copied the 88 from the `flake8` configuration, and I'm not sure where that came from  :)

Okay, I'll switch both to 80.